CRM/Financeextras/Upgrader.php

```diff
@@ -162,4 +162,31 @@ public function upgrade_1002() {
     return TRUE;
   }
 
+  /**
+   * This upgrade updates line items that have empty price field values
+   */
+  public function upgrade_1003() {
```

Comment:
Why is the upgrader necessary @shahrukh-compuco, can you explain this in the PR?

Reply:
donr